lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -82,6 +82,8 @@
 <template id="noopener-links-tmpl">
   <!-- FAIL - does not use rel="noopener" to open external link -->
   <a href="https://www.google.com/" target="_blank">external link</a>
+  <!-- FAIL - does not use rel="noopener" but no href attribute -->
+  <a target="_blank">external link</a>
```

Comment:
maybe add a comment here or in the audit that when no href is given, the browser has a '' by default. Thats why you do the '' check I suppose.